seaice_ecdr/temporal_composite_daily.py

```diff
@@ -442,6 +407,40 @@ def temporally_interpolated_ecdr_dataset_for_au_si_tbs(
         },
     )
 
+    if fill_the_pole_hole:
+        cdr_conc = np.squeeze(tie_ds["cdr_conc_ti"].data)
+        tie_ds["cdr_conc"] = tie_ds["cdr_conc_ti"].copy()
+        # TODO: This is a really coarse way of determining which
+        #       grid is having its pole hole filled!
+
+        # TODO: These are constants for the eCDR runs.  They should
+        #       NOT be definied here (and in the idecdr code...(!))
+        tb_spatint_bitmask_map = {
+            "v18": 1,
+            "h18": 2,
+            "v23": 4,
+            "v36": 8,
+            "h36": 16,
+            "pole_filled": 32,
+        }
+        if cdr_conc.shape == (896, 608):
```

Comment:
```suggestion
        if hemisphere == NORTH:
```
Use hemisphere here instead of shape.

Reply:
I addressed this with the prior code reworking.  This code is specifically designed for psn12.5, so will not work with all NH grids.  (It reads the psn12.5 polehole mask)